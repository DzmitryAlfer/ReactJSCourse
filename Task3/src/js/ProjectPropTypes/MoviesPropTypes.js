import PropType from 'prop-types'
import {MoviePropType} from './MoviePropType';

export const MoviesPropType = {
    data: PropType.arrayOf(PropType.shape(MoviePropType)),
    total: PropType.number.isRequired,
    offset: PropType.number.isRequired,
    limit: PropType.number.isRequired
};