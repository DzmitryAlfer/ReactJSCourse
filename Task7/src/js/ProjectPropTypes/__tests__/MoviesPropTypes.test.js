import {MoviesPropTypes} from "../MoviesPropTypes";

describe('movie array prop type checker', () => {
    test('movie array prop type checker', () => {
        expect(MoviesPropTypes).toMatchSnapshot();
    });
});