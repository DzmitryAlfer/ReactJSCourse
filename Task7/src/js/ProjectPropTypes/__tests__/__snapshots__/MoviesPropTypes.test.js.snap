// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`movie array prop type checker movie array prop type checker 1`] = `
Object {
  "data": [Function],
  "limit": [Function],
  "offset": [Function],
  "total": [Function],
}
`;
